octoprint_mrbeam/static/js/mrbeam.js

```diff
@@ -390,8 +391,10 @@ $(function () {
                     settings,
                     thrownError
                 ) {
-                    if (jqXHR.status == 401) {
-                        self._handle_session_expired(settings.url);
+                    if (jqXHR.status === 401) {
+                        // "self.loginState.loggedIn()" sometimes returns true when the user is actually logged out
+                        console.log("401 error - data:", settings.data, " reponse:", jqXHR.responseText, " loggedin:", self.loginState.loggedIn(), " loginrember:", self.loginState.loginRemember())
+                        self._handle_session_expired(settings.url, settings.data);
```

Comment:
Please place this before console logging the data as there is a possibility that one of these logged values may return an exception. this way the session handling will still be working.